pkg/integration/docker.go

```diff
@@ -0,0 +1,150 @@
+package integration
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/dhis2-sre/im-manager/pkg/config"
+)
+
+func NewDockerHubClient(config config.Config) dockerHubClient {
```

Comment:
These are the only things

```go
username := d.config.DockerHub.Username
password := d.config.DockerHub.Password
```

the `DockerHubClient` needs. So Username and Password should be the only parameters passed in and fields stored.

Reply:
You always get me on this one! Thanks :-)